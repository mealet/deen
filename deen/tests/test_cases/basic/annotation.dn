fn main() i32 {
  // identifier test
  let _some_variable_withStrangeNAME123 = 10;
  println!("{}", _some_variable_withStrangeNAME123);

  // redefinition test
  let var = "hello";
  println!("{}", var);

  let var = 10;
  println!("{}", var);

  // empty annotation
  let _ = 10;

  return 0;
}
