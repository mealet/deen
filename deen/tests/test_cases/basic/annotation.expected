10
hello
10
